src/System.IO.StreamSourceGeneration/src/BoilerplateCandidateInfo.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Constants = System.IO.StreamSourceGeneration.StreamBoilerplateConstants;
+
+namespace System.IO.StreamSourceGeneration;
```

Comment:
I see 122 matches of `namespace [A-Za-z0-9.]*;` using VSCode search. 

Just to name one: https://github.com/dotnet/runtime/blob/53bb518ea482a5960577b151c11cc1297e5dea6d/src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/GeneratedRegexAttribute.cs#L8

Reply:
> Just to name one

That's... surprising.  I must have done that accidentally, since a) we're not changing existing usage and b) our general approach is to match the style used by the rest of the project.